spectral_cube/spectral_cube.py

```diff
@@ -81,7 +80,7 @@ def wrapper(*args, **kwargs):
 # conventions between WCS and numpy
 np2wcs = {2: 0, 1: 1, 0: 2}
 
-class BaseSpectralCube(BaseNDClass, SpectralAxisMixinClass):
+class BaseSpectralCube(BaseNDClass, SpectralAxisMixinClass, SpatialCoordMixinClass):
```

Comment:
I just learned recently... I think `super` always refers to the *last* of these 3, and so maybe we want `BaseNDClass` last?  But we should check with someone who knows better (...or rtfd...) before changing it